e2e/test/scenarios/question/saved.cy.spec.js

```diff
@@ -251,44 +251,32 @@ describe("scenarios > question > saved", () => {
     });
   });
 
-  it(
-    "'read-only' user should be able to resize column width (metabase#9772)",
-    { tags: "@flaky" },
-    () => {
-      cy.signIn("readonly");
-      visitQuestion(ORDERS_QUESTION_ID);
-
-      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.findByText("Tax")
-        .closest(".test-TableInteractive-headerCellData")
-        .as("headerCell")
-        .then($cell => {
-          const originalWidth = $cell[0].getBoundingClientRect().width;
-
-          // Retries the assertion a few times to ensure it waits for DOM changes
-          // More context: https://github.com/metabase/metabase/pull/21823#discussion_r855302036
-          function assertColumnResized(attempt = 0) {
-            cy.get("@headerCell").then($newCell => {
-              const newWidth = $newCell[0].getBoundingClientRect().width;
-              if (newWidth === originalWidth && attempt < 3) {
-                cy.wait(100);
-                assertColumnResized(++attempt);
-              } else {
-                expect(newWidth).to.be.gt(originalWidth);
-              }
-            });
-          }
-
-          cy.wrap($cell)
-            .find(".react-draggable")
-            .trigger("mousedown", 0, 0, { force: true })
-            .trigger("mousemove", 100, 0, { force: true })
-            .trigger("mouseup", 100, 0, { force: true });
-
-          assertColumnResized();
-        });
-    },
-  );
+  it("'read-only' user should be able to resize column width (metabase#9772)", () => {
+    cy.signIn("readonly");
+    visitQuestion(ORDERS_QUESTION_ID);
+
+    cy.findAllByTestId("header-cell")
+      .contains("Tax")
+      .closest(".test-TableInteractive-headerCellData")
+      .as("headerCell")
```

Comment:
Let's fix this Frankenstein.

```suggestion
    cy.findAllByTestId("header-cell")
      .filter(":contains(Tax)")
      .as("headerCell")
```

You can also try something even simpler (but not sure if it's going to work):
`cy.findByLabel("Tax").as("headerCell");`

Reply:
we need that parent as `headerCell` afaik, but I'll try your suggestion

Reply:
![image](https://github.com/user-attachments/assets/ff34a1ed-1dff-4d85-b763-84518c2362bd)

Reply:
it worked, thanks!